src/providers/SpeechProvider/SpeechProvider.container.js

```diff
@@ -48,10 +50,12 @@ export class SpeechProvider extends Component {
       } catch (err) {
         console.error(err.message);
       }
+      this.setState({ voicesRecived: true });
     }
   }
 
   render() {
+    if (!this.state.voicesRecived) return null;
```

Comment:
@tomivm what would happen if the `await getVoices()` sentence does never return? Remember that js async calls don't have a default timeout, and getting voices depends on the platform. looks risky to render blank, it will Break the rendering chain of the application.

Reply:
thanks for your review! I will avoid it